README.md

```diff
@@ -1,5 +1,6 @@
 # Loopchain
 
+[![loopchain](https://img.shields.io/badge/ICON-Consensus-blue?logoColor=white&logo=icon&labelColor=31B8BB)](https://shields.io)
```

Comment:
This PR is already merged. I have a few suggestions.
- It would be better change `https://shields.io` to icon related url. For example `iconloop.com`.
- Why don't you rename `ICON-loopchain` instead of `ICON-Consensus`?

Reply:
It has following format:
[Label]-[Message]-[Color]
> ICON-Consensus

I think he intended that icon-project as ICON, and locate Team name to message.